### ISSUE TITLE
Builder can't call howl

### ISSUE THREAD

**Issue by Lupus590** [CC-Hive/Main#11]
https://squiddev-cc.github.io/howl.ci/?p=travis/build&id=179600065

https://squiddev-cc.github.io/howl.ci/?p=travis/build&id=180241147

**Comment by SquidDev** [comment]
Yeah, you can't specify a Howlfile on the command line. Instead do:

```lua
shell.setDir("Code")
if not shell.run("/Howl.min.lua") then
    howlci.status("error","failed to call Howl")
end
howlci.close()
```

A couple of other things:
 - I'm using `error` instead of `fail` as this means something broke with the build process, rather than a test failing.
 - `howlci.close()` means the build ends straight away, rather than waiting for the computer to terminate fully.

**Comment by Lupus590** [comment]
still no worky :( https://squiddev-cc.github.io/howl.ci/?p=travis/build&id=181325208

~~could it be that the Howl file is empty? (I think it's empty)~~ `
if howlci then howlci.status("ok","nothing in /Code/HowlFile.lua yet") end`

**Comment by SquidDev** [comment]
Ahh, it should be `Howlfile.lua`: note capitalisation. I should probably make it case insensitive :/.

**Comment by SquidDev** [comment]
Fixed in 6a4f7c3d843b7a27be82262d5d48d53c455da6aa. I forgot I had push access.
